app/Platform/Controllers/RelatedGamesTableController.php

```diff
@@ -54,23 +58,53 @@ public function __invoke(Request $request): View
             });
         }
 
+        if (isset($filterOptions['status'])) {
+            $this->gameListService->filterGameList(function ($game) use ($filterOptions) {
+                return $this->gameListService->useGameStatusFilter($game, $filterOptions['status']);
+            });
+        }
+
         $this->gameListService->mergeWantToPlay($loggedInUser);
 
         $this->gameListService->sortGameList($sortOrder);
 
         $availableSorts = $this->gameListService->getAvailableSorts();
-        $availableFilters = [
+        $availableCheckboxFilters = [
             'console' => 'Group by console',
             'populated' => 'Only with achievements',
         ];
+        $availableSelectFilters = [
+            [
+                'kind' => 'status',
+                'label' => 'Status',
+                'options' => [
+                    'all' => 'All games',
+                    'unstarted' => 'No progress',
+                    'unawarded' => 'Unfinished',
```

Comment:
Should `unawarded` also return `unstarted`? Or maybe the label could be clarified?

Reply:
In the latest commit, I've tried to make the filter options a little more clear:

![Screenshot 2024-01-14 at 11 08 41 AM](https://github.com/RetroAchievements/RAWeb/assets/3984985/13dc8cb8-c960-41b9-8743-f1fa618bf9db)

`unawarded` internally has been renamed to `lt-beaten-softcore`. The label has changed to "Has progress, but no award".